exercises/APE11/1101.tex

```diff
@@ -0,0 +1,19 @@
+\begin{itemize}
+  \item[a.] The decision that we will do during the first turn of the prisoner dilemma depends of the one done during the second turn,
+  that depends of the one of the third turn,etc. As we have a finite number of games, we will check the rationnal decison done at the last turn,
+  the game $N$.
+  In this case, the past has no impact and the players will both defect $(D,d)$. Now that we know that, we can done a decision $(D,d)$
```

Comment:
"Now that we know that, we can done a decision $(D,d)$ for the game $N-1$."
It is not clear what you mean by that. Say explicitly that since the choice made on the last round does not depend on the round N-1, the player will play round N-1 just like it was the last round.